pom.xml

```diff
@@ -96,6 +96,12 @@
         <at-symbol>@</at-symbol>
         <variable-prefix>${</variable-prefix>
         <image-width>500</image-width>
+        <deprecated>
+[IMPORTANT]
```

Comment:
❓ Do the new lines before and after matter?

Reply:
Yes. it's getting inserted into asciidoctor content, which cares about the whitespace